### PR TITLE
Fixed problem with returning false if cookiePath end with '/'

### PR DESCRIPTION
**Pull request by QweKs-dev**
Now
`urlPathLowerCase = /touch/rest/booking/suggested-seating/-17727`
`cookiePathLowerCase = /touch/rest/`
`temp = /booking/suggested-seating/-17727`
And returning **true**

### CODE REVIEW

**Comment by QweKs-dev** [comment]
Currently `return temp.startsWith('/')` does not work correctly and returns false even if the string should match. The slash character is truncated at the beginning of the line when calculating the temp.

**Comment by AlexV525** [comment]
> The tests nested in the package passed successfully.

That's great, but not enough. We should consider adding tests that fail with previous commits and are fixed by this change. More specifically, tests should ensure that the path in Cookies is resolved correctly by constructing Cookies and verifying them.

**Comment by QweKs-dev** [comment]
Can you tell me how long this will take?

**Comment by AlexV525** [comment]
> Can you tell me how long this will take?

I'm not sure what you're indicating about "how long". The only missing thing is to add tests for the change.

File: lib/src/default_cookie_jar.dart [hunk 1]
```diff
@@ -59,7 +59,9 @@ class DefaultCookieJar implements CookieJar {
       return true;
     }
     if (urlPathLowerCase.startsWith(cookiePathLowerCase)) {
-      final temp = urlPathLowerCase.substring(cookiePathLowerCase.length);
+      final temp = urlPathLowerCase
+          .substring(cookiePathLowerCase.endsWith('/') ?
+      cookiePathLowerCase.length - 1 : cookiePathLowerCase.length,);
```
Comment (AlexV525) [inline lib/src/default_cookie_jar.dart:64]: Also, the code does not seem to be well formatted.

Comment (QweKs-dev) [reply lib/src/default_cookie_jar.dart:64]: Who will add tests You or I?
And how does this happen?

Comment (AlexV525) [reply lib/src/default_cookie_jar.dart:64]: > Who will add tests You or I?

Well. The PR author for sure.

> And how does this happen?

Try `dart format` before the next commit.

**Comment by BenderBlog** [comment]
Just a note here:

**Pull request merged**